### ISSUE TITLE
Default block break sound effect

### ISSUE THREAD

**Issue by deathcap** [deathcap/voxel-sfx#11]
https://github.com/deathcap/voxel-sfx/pull/10 added playing the 'block/break' sound on block break but this is not a known sound effect name in:

* https://github.com/deathcap/ProgrammerArt ([source](https://github.com/deathcap/ProgrammerArt/blob/master/scripts/generate.js#L57)) - default voxel-engine-stackgl / voxel-stitch art pack
* Another popular 3rd party sound pack, http://bdcraft.net/bdcraft-sounds-pack

**Comment by deathcap** [comment]
@z3t0 Ideally the default would be an existing sound resource name present in Minecraft-style resource packs (http://minecraft.gamepedia.com/Resource_pack), so that 3rd party packs can be used to replace the sound. ProgrammerArt for example is released in Minecraft resource pack format (and also, "texture pack" and "stitch pack" formats but they aren't used in voxel.js), read by https://github.com/deathcap/artpacks, so it can be easily switched out with custom replacement packs.

**Comment by z3t0** [comment]
@deathcap I see, do minecraft resource packs have sound built-in? If so we can just adapt that.

**Comment by deathcap** [comment]
Yeah that's what I was thinking. Question is, which sound effect name? `random/pop`?

**Comment by z3t0** [comment]
random/block is the default

--
Thanks,
Rafi Khan 

----- Reply message -----
